templates/org.html

```diff
@@ -6,7 +6,9 @@
 {{ define "canonical" }}/{{ include "helpers/get-country-url.html" .country }}/{{ .qid }}/{{ end }}
 
 {{ define "extra-head" }}
-  <meta name="description" content="{{ .orgDescription }}">
+    {{- if .orgDescription -}}
```

Comment:
Do we want a generic fallback to something like "Organization in Govdirectory" if this is false or does something like that already happen?

Reply:
@Ainali  I think that would make it more likely to get flagged for "duplicate content", a future to do could be to make it an unique sentence of some kind, but then it might just be a better effort to work on descriptions in Wikidata.